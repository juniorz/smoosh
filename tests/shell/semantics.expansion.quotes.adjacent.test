mkdir a
touch a/b
touch a/c
echo a/*
echo "a"/*
echo 'a'/*
mkdir "foo*["
touch "foo*["/weird
touch "foo*["/wild
touch "foo*["/crazy
echo "foo*["/*
echo "foo*["/[wz]*