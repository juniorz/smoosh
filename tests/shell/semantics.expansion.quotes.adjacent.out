a/b a/c
a/b a/c
a/b a/c
foo*[/crazy foo*[/weird foo*[/wild
foo*[/weird foo*[/wild
